recipes/ldas-tools-framecpp-swig/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "ldas-tools-framecpp-swig" %}
+{% set version = "2.6.7" %}
+{% set sha256 = "d9a0c2626f0cd9b066c875232afa79febd1e4cbc5fb0103830e780429256ab52" %}
+{% set ldas_tools_cmake_version = "1.0.6" %}
+{% set ldas_tools_al_version = "2.6.2" %}
+{% set ldas_tools_framecpp_version = "2.6.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.6
+    - make  # [not win]
+    - pkg-config  # [not win]
+    - swig
+  host:
+    - ldas-tools-cmake >={{ ldas_tools_cmake_version }}
+    - ldas-tools-framecpp >={{ ldas_tools_framecpp_version }}
+  run:
+    - ldas-tools-framecpp >={{ ldas_tools_framecpp_version }}
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/framecpp/frameCPP.i  # [not win]
+    - test -f ${PREFIX}/include/framecpp/python/frameCPPMacros.i  # [not win]
+
+outputs:
+  - name: ldas-tools-framecpp-swig
```

Comment:
Wow. I have not seen this incantation of multiple outputs before. Kudos!

Reply:
This was stolen mercilessly from conda-forge/matplotlib-feedstock#196, its neat.

Reply:
@beckermr I am ok being a maintainer of this package.